code/game/objects/items/melee/baton.dm

```diff
@@ -519,9 +519,12 @@
 
 /obj/item/melee/baton/security/attack_self(mob/user)
 	if(cell?.charge >= cell_hit_cost)
-		active = !active
-		balloon_alert(user, "turned [active ? "on" : "off"]")
-		playsound(src, SFX_SPARKS, 75, TRUE, -1)
+		if(SSsecurity_level.get_current_level_as_number() >= SEC_LEVEL_RED)
+			active = !active
+			balloon_alert(user, "turned [active ? "on" : "off"]")
+			playsound(src, SFX_SPARKS, 75, TRUE, -1)
+		else:
```

Comment:
```suggestion
		else
```

?

Reply:
pythonbros...

Reply:
YO UCAN'T FORCE PUSH OVER YOUR CHANGES WE ALL SAW THIS